todo-app/src/App.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef, useState } from "react";
+import styles from './App.module.css'
+
+
+function App() {
+
+  const [toDo, setToDo] = useState('');
+  const [toDos, setToDos] = useState([]);
```

Comment:
네이밍때문에 헷갈릴수 있습니다.
좀 더 명시적인 이름을 쓰는 것이 좋을 것 같습니다.

```suggestion
const [inputValue, setInputValue] = useState('');
const [toDoItems, setToDoItems] = useState([]);
```